pcomfortcloud/constants.py

```diff
@@ -1,5 +1,13 @@
 from enum import Enum
 
+APP_CLIENT_ID = "Xmy6xIYIitMxngjB2rHvlm6HSDNnaMJx"
+AUTH_0_CLIENT = "eyJuYW1lIjoiQXV0aDAuQW5kcm9pZCIsImVudiI6eyJhbmRyb2lkIjoiMzAifSwidmVyc2lvbiI6IjIuOS4zIn0="
+REDIRECT_URI = "panasonic-iot-cfc://authglb.digital.panasonic.com/android/com.panasonic.ACCsmart/callback"
+BASE_PATH_AUTH = "https://authglb.digital.panasonic.com"
+BASE_PATH_ACC = "https://accsmart.panasonic.com"
+X_APP_VERSION = "1.21.0"
```

Comment:
this is something you're going to have to bump every now and then.. the scraping via appbrain has had issues randomly (app brain going down or not updating it fast enough)

so be aware that at worst the app version will be wrong until where it scrapes from updates. So a couple hours or so. But this is only a concern when you're trying to re-auth/get a new token

Reply:
we could store this in a file as well maybe